mithril-stm/src/schnorr_signature/mod.rs

```diff
@@ -0,0 +1,168 @@
+#![allow(dead_code)]
+
+use midnight_circuits::{
+    ecc::{hash_to_curve::HashToCurveGadget, native::EccChip},
+    hash::poseidon::PoseidonChip,
+    types::AssignedNative,
+};
+use midnight_curves::{
+    Fq as JubjubBase, Fr as JubjubScalar, JubjubAffine, JubjubExtended, JubjubSubgroup,
+};
+use sha2::{Digest, Sha256};
+
+use anyhow::{Result, anyhow};
+
+mod signature;
+mod signing_key;
+mod verification_key;
+
+/// A DST to distinguish between use of Poseidon hash
```

Comment:
```suggestion

use signature::*;
use signing_key::*;
use verification_key::*;

/// A DST to distinguish between use of Poseidon hash
```
By doing so, we can simplify the imports in the files. In the future, we might make these uses public if needed.